rsyslog/service_files/rsyslog.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=System Logging Service
+Requires=syslog.socket
+Documentation=man:rsyslogd(8)
+Documentation=https://www.rsyslog.com/doc/
+
+[Service]
+Type=notify
+ExecStart=/usr/sbin/rsyslogd -n -iNONE
```

Comment:
Just curious why is the pid file set to NONE?
https://linux.die.net/man/8/rsyslogd

Reply:
This was ripped from the official Rsyslog git repo and it works like a charm